### PR TITLE
Ft/desktop version

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, please review my project and advice. Thanks!

### Changes made:

- Add Work section
- Add About me section
- Add Skills section
- Add Contact me section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1503,3 +1507,435 @@
   font-size: 23px;
   margin-left: 18px;
 }
+
+@media only screen and (min-width: 768px) {
```
Comment (emmiiorji) [inline style.css:1532]: You have done great implementing the desktop version of the portfolio. Kindly ensure that the mobile version is responsive until the desktop screen size (768px or larger).

Your design at 558px:
image

File: style.css [hunk 2]
```diff
@@ -1503,3 +1507,435 @@
...
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .headline {
+    width: 100vw;
+    height: 609px;
+    flex-direction: row;
+    background-image: url(./imgs/GroupBig.svg);
+  }
```
Comment (emmiiorji) [inline style.css:1582]: Kindly ensure that the background image does not overlap with the text.

Your design at 768px:
image

File: style.css [hunk 3]
```diff
@@ -1503,3 +1507,435 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 130px 24px;
+    gap: 114px;
+    position: absolute;
+    width: 100vw;
+    height: 1805px;
+    top: 875px;
+  }
```
Comment (emmiiorji) [inline style.css]: You have done great at implementing the project cards. A closer look at the figma design shows that there're two form that the project cards take. One is the normal display of the card without the "See Project" button, the other is the way each card is expected to look when hovered where the "See Project" button appears. Kindly implement this too in order to follow the design.

Card's normal display:
image

Card when hovered:
image

File: style.css [hunk 4]
```diff
@@ -1503,3 +1507,435 @@
...
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 130px 0;
+    gap: 152px;
+    position: absolute;
+    width: 100vw;
+    height: 596px;
+    left: 141px;
+    top: 629px;
+  }
```
Comment (emmiiorji) [inline style.css:1858]: Kindly ensure that the skills section follows the figma design as closely as possible. Consider how the list items are vertically aligned with the headings. Also ensure that the background color matches that of the design

Your design:
image

Figma design:
image

File: style.css [hunk 5]
```diff
@@ -1503,3 +1507,435 @@
...
+
+  .nav-menu {
+    display: flex;
+    list-style-type: none;
+  }
+
+  .nav-menu li {
+    color: #fff;
+    margin-left: 80px;
+  }
```
Comment (emmiiorji) [inline style.css]: For a better user experience on the website, kindly ensure that the content fits on the page without overflow.

Your design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @egichuhis, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (egichuhis) [reply style.css]: Done.

File: style.css (hunk 4 above)
Comment (egichuhis) [reply style.css:1858]: Done.

File: style.css (hunk 2 above)
Comment (egichuhis) [reply style.css:1582]: Done.

File: style.css (hunk 5 above)
Comment (egichuhis) [reply style.css]: Done.

File: style.css (hunk 1 above)
Comment (egichuhis) [reply style.css:1532]: Done.

File: index.html [hunk 6]
```diff
@@ -17,24 +17,36 @@
...
-            <h2 class="supporting">
-                I’m a Software Developer
-            </h2>
-            <p class="description">
-                I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you
-                see and have a project you need coded,
-                don’t hesitate to contact me.
-            </p>
+            <span class="headline-text">
```
Comment (huseyinbiyik) [inline index.html:34]: As the previous reviewer requested, please fix the overlaying issue on the heading section 👍
Screenshot 2023-01-28 at 11 15 01 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @egichuhis,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Card hovers implemented very well!
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝


 ## Required Changes ♻️
- As the previous reviewer requested, please fix the alignment on the language section like you did Frameworks and Skills.
Screenshot 2023-01-28 at 11 13 04 PM

_Check the comments under the review._

## Optional suggestions
[OPTIONAL] Please make sure your webpage is responsive for all sizes, you can see how it is breaking over 768px. 
Screen Recording 2023-01-28 at 11 16 05 PM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (egichuhis) [reply index.html:34]: I have fixed the required changes. I will work on the Optional changes after the submission due to the deadline approaching in a few minutes. Making it fully responsive will take me some more time and I don't want to miss the deadline and repeat the week. Thanks.

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — COMMENTED** [review]
Hi @egichuhis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good level of responsiveness :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**